From 7d660d2f675db1072e71f9138822bd354a5ca88c Mon Sep 17 00:00:00 2001
Message-ID: <7d660d2f675db1072e71f9138822bd354a5ca88c.1705743628.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 17 Jan 2024 21:38:38 +0000
Subject: [PATCH 32/38] gnu: go-filippo-io-age: Move to (gnu packages
 golang-crypto).

* gnu/packages/golang.scm (go-filippo-io-age): Move from here ...
* gnu/packages/golang-crypto.scm: ... to here.

Change-Id: I4eeadfd946910d56d2441a3469967d681b9e8677
---
 gnu/packages/golang-crypto.scm | 32 +++++++++++++++++++++++++++++++-
 gnu/packages/golang.scm        | 29 -----------------------------
 2 files changed, 31 insertions(+), 30 deletions(-)

diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index 9c9e3d26b8..7dede78874 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -11,11 +11,12 @@
 ;;; Copyright © 2021 Vagrant Cascadian <vagrant@debian.org>
 ;;; Copyright © 2022 (unmatched-parenthesis <paren@disroot.org>
 ;;; Copyright © 2022 Efraim Flashner <efraim@flashner.co.il>
+;;; Copyright © 2022 Tobias Geerinckx-Rice <me@tobias.gr>
 ;;; Copyright © 2022, 2023 Nicolas Graves <ngraves@ngraves.fr>
 ;;; Copyright © 2023 Artyom V. Poptsov <poptsov.artyom@gmail.com>
 ;;; Copyright © 2023 Clément Lassieur <clement@lassieur.org>
 ;;; Copyright © 2023 Felix Lechner <felix.lechner@lease-up.com>
-;;; Copyright © 2023 Nicolas Graves <ngraves@ngraves.fr>
+;;; Copyright © 2023 Jack Hill <jackhill@jackhill.us>
 ;;;
 ;;; This file is part of GNU Guix.
 ;;;
@@ -53,6 +54,35 @@ (define-module (gnu packages golang-crypto)
 ;;;
 ;;; Code:
 
+(define-public go-filippo-io-age
+  (package
+    (name "go-filippo-io-age")
+    (version "1.1.1")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/FiloSottile/age")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1k1dv1jkr72qpk5g363mhrg9hnf5c9qgv4l16l13m4yh08jp271d"))))
+    (build-system go-build-system)
+    (arguments `(#:import-path "filippo.io/age"))
+    (inputs
+     (list go-golang-org-x-sys
+           go-golang-org-x-term
+           go-golang-org-x-crypto
+           go-filippo-io-edwards25519))
+    (home-page "https://filippo.io/age")
+    (synopsis "Secure file encryption tool, format, and Go library")
+    (description
+     "This package implements file encryption according to the
+@{age-encryption.org/v1, https://age-encryption.org/v1} specification.
+It features small explicit keys, no configuration options, and Unix-style
+composability.")
+    (license license:bsd-3)))
+
 (define-public go-filippo-io-edwards25519
   (package
     (name "go-filippo-io-edwards25519")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index d79b351498..245de0dc32 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -9375,35 +9375,6 @@ (define-public go-etcd-io-bbolt
     (description "This package implements a low-level key/value store in Go.")
     (license license:expat)))
 
-(define-public go-filippo-io-age
-  (package
-    (name "go-filippo-io-age")
-    (version "1.1.1")
-    (source
-     (origin
-       (method git-fetch)
-       (uri (git-reference
-             (url "https://github.com/FiloSottile/age")
-             (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32 "1k1dv1jkr72qpk5g363mhrg9hnf5c9qgv4l16l13m4yh08jp271d"))))
-    (build-system go-build-system)
-    (arguments `(#:import-path "filippo.io/age"))
-    (inputs
-     (list go-golang-org-x-sys
-           go-golang-org-x-term
-           go-golang-org-x-crypto
-           go-filippo-io-edwards25519))
-    (home-page "https://filippo.io/age")
-    (synopsis "Secure file encryption tool, format, and Go library")
-    (description
-     "This package implements file encryption according to the
-@{age-encryption.org/v1, https://age-encryption.org/v1} specification.
-It features small explicit keys, no configuration options, and Unix-style
-composability.")
-    (license license:bsd-3)))
-
 (define-public age
   (package
     (inherit go-filippo-io-age)
-- 
2.41.0

